# HG changeset patch
# User Jordi Gutiérrez Hermoso <jordigh@gmail.com>
# Date 1301512073 21600
#      Wed Mar 30 13:07:53 2011 -0600
# Node ID 1cfa3d9adf0ab3f4e264cf0a8e2ed874c8bdf8a2
# Parent  e4a1ede4e832a03dd8acba92f7c9a72ee655320f
Document the "all" parameter to fclose.

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2011-03-30  Jordi Gutiérrez Hermoso  <jordigh@gmail.com>
+
+	* file-io.cc: Document the "all" parameter to fclose.
+
 2011-03-29  Ben Abbott  <bpabbott@mac.com>
 
 	* graphics.cc: Prevent ticks from being cropped by round-off errors
 	when limmode == auto (bug #32701).
 
 2011-03-28  Rik  <octave@nomad.inbox5.com>
 
 	* DLD-FUNCTIONS/inv.cc (inv, inverse), DLD-FUNCTIONS/tril.cc (tril),
diff --git a/src/file-io.cc b/src/file-io.cc
--- a/src/file-io.cc
+++ b/src/file-io.cc
@@ -205,18 +205,20 @@ fopen_mode_to_ios_mode (const std::strin
     }
 
   return retval;
 }
 
 DEFUN (fclose, args, ,
   "-*- texinfo -*-\n\
 @deftypefn {Built-in Function} {} fclose (@var{fid})\n\
+@deftypefnx {Built-in Function} {} fclose (\"all\")\n\
 Close the specified file.  If successful, @code{fclose} returns 0,\n\
-otherwise, it returns -1.\n\
+otherwise, it returns -1. The second form of the @code{fclose} call closes\n\
+all open files except @code{stdout}, @code{stderr}, and @code{stdin}.\n\
 @seealso{fopen, fseek, ftell}\n\
 @end deftypefn")
 {
   octave_value retval = -1;
 
   int nargin = args.length ();
 
   if (nargin == 1)
